Common_Words_Scraping.py

```diff
@@ -0,0 +1,45 @@
+# coding: utf-8
+import time
+from lxml import html
+import requests
+import re
+from bs4 import BeautifulSoup
+import os
+import csv
+from Wiktionary_scraper import *
+
+def common_words(language):
+    """
+    function that will give us the 1000 most common words in a language from http://1000mostcommonwords.com
+    """
+    url = 'http://1000mostcommonwords.com/1000-most-common-%s-words/' %language
+    try:
+        request = requests.get(url).content
+    except:
```

Comment:
since you are catching the exception, it's good to log it. you will understand what went wrong when it happens. I would suggest:
```
except Exception as e:
    logging.error("Error while fetching {0}:\n{1}".format(url, e))
```